docs/architecture/cloud-native/map-eshoponcontainers-azure-services.md

```diff
@@ -1,13 +1,69 @@
 ---
 title: Mapping eShopOnContainers to Azure Services
-description: Architecting Cloud Native .NET Apps for Azure | Mapping eShopOnContainers to Azure Services
+description: Architecting Cloud-native .NET Apps for Azure | Mapping eShopOnContainers to Azure Services
 ms.date: 06/30/2019
 ---
 # Mapping eShopOnContainers to Azure Services
 
-Although not required, Azure is very well-suited to supporting the eShopOnContainers because the project was built to be a cloud native application.
+Although not required, Azure is very well-suited to supporting the eShopOnContainers because the project was built to be a cloud-native application. The application is built with .NET Core, so it's capable of running on Linux or Windows containers depending on the Docker host. The application is made up of multiple autonomous microservices, each with its own data. The different microservices showcase different approaches, ranging from simple CRUD operations to more complex DDD and CQRS patterns. Microservices communicate with clients over HTTP and with one another via message-based communication. The application supports multiple platforms for clients as well, since it leverages HTTP as a standard communication protocol and includes ASP.NET Core apps as well as Xamarin mobile apps that run on Android, iOS, and Windows platforms.
```

Comment:
```suggestion
Although not required, Azure is well-suited to supporting the eShopOnContainers because the project was built to be a cloud-native application. The application is built with .NET Core, so it's capable of running on Linux or Windows containers depending on the Docker host. The application is made up of multiple autonomous microservices, each with its own data. The different microservices showcase different approaches, ranging from simple CRUD operations to more complex DDD and CQRS patterns. Microservices communicate with clients over HTTP and with one another via message-based communication. The application supports multiple platforms for clients as well, since it adopts HTTP as a standard communication protocol and includes ASP.NET Core apps as well as Xamarin mobile apps that run on Android, iOS, and Windows platforms.
```